src/go/rpk/pkg/cli/cmd/cluster/txn/abort.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package txn
+
+import (
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+)
+
+func newAbortCommand(fs afero.Fs) *cobra.Command {
+	var (
+		topic       string
+		partition   int32
+		startOffset int64
+	)
+	cmd := &cobra.Command{
+		Use:   "abort",
+		Short: "Abort an open transaction for a given partition",
+		Long: `Abort an open transaction for a given partition.
+
+This command aborts a single partition in a transaction, allowing the
+last-stable-offset to advance for the partition. To be extra cautious, this
+command only aborts one partition at a time. You can run this repeatedly to
+abort all partitions that may be hanging in a single transaction.
+
+You can find potentially hanging transactions with the find-hanging command.
+
```

Comment:
[nit] Not needed, Cobra already adds a new line at the end of the usage string.

Reply:
Good nit, not sure how this got here